app/src/main/res/values/strings.xml

```diff
@@ -1211,7 +1211,7 @@
     <string name="logins_password_copied">Password copied to clipboard</string>
     <!-- Shown in snackbar to tell user that the username has been copied -->
     <string name="logins_username_copied">Username copied to clipboard</string>
-    <!-- Shown in snackbar to tell user that the site has been copied -->
+    <!-- Shown in snackbar to tell user that the web address has been copied -->
```

Comment:
same l10n issue here I believe. I don't think we can change the description after it has been exported :/